audio/whisper/Portfile

```diff
@@ -0,0 +1,120 @@
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup   github      1.0
+PortGroup   makefile    1.0
```

Comment:
please align the lunes above

Reply:
@reneeotten Can you explain how these lines should be aligned? Should there just be one tab between each item, or should they visually align? See two images:
<img width="384" alt="Screenshot 2024-08-16 at 9 34 28 PM" src="https://github.com/user-attachments/assets/e6df899c-67b7-401c-90be-c602ccb712b4">
(1 tab between each item)
Or
<img width="348" alt="Screenshot 2024-08-16 at 9 35 02 PM" src="https://github.com/user-attachments/assets/a6c4ad62-9a51-42e2-8b32-f5ca3da182cc">
(items align visually)

Reply:
Oh, having reviewed some other Portfiles, I think I see how it should be.